src/store.ts

```diff
@@ -37,6 +37,23 @@ function supportsLocalStorage() {
   }
 }
 
+/**
+ * Checks if the given object property is accessible.
+ * Ensures that the property exists and that accessing it
+ * does not throw
+ */
+function isPropertyAccessible<Obj extends Record<string, any>>(
+  object: Obj,
+  method: keyof Obj,
```

Comment:
scoped this to properties defined on the type, which seems to be narrow enough here